<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-A2S9SMV" failures="0" tests="7" name="com.oneapp.test_suits.EmergencyContact_testcases" time="760.526" errors="0" timestamp="2023-03-10T13:44:08 IST" skipped="0">
  <testcase classname="com.oneapp.test_suits.EmergencyContact_testcases" name="validate_emergency_contact_added_succesfully_testcase" time="224.204"/>
  <system-out/>
  <testcase classname="com.oneapp.test_suits.EmergencyContact_testcases" name="validate_emergency_contact_deleted_succesfully_testcase" time="60.474"/>
  <system-out/>
  <testcase classname="com.oneapp.test_suits.EmergencyContact_testcases" name="validate_emergency_contact_with_blank_fullName_testcase" time="163.532"/>
  <system-out/>
  <testcase classname="com.oneapp.test_suits.EmergencyContact_testcases" name="validate_emergency_contact_with_blank_MobileNumber_testcase" time="36.206"/>
  <system-out/>
  <testcase classname="com.oneapp.test_suits.EmergencyContact_testcases" name="validate_emergency_contact_with_blank_relationType_testcase" time="36.604"/>
  <system-out/>
  <testcase classname="com.oneapp.test_suits.EmergencyContact_testcases" name="validate_emergency_contact_saving_with_invalid_details_testcase" time="59.954"/>
  <system-out/>
  <testcase classname="com.oneapp.test_suits.EmergencyContact_testcases" name="validate_emergency_contact_already_exist_testcase" time="179.552"/>
  <system-out/>
</testsuite> <!-- com.oneapp.test_suits.EmergencyContact_testcases -->
